examples/TUTORIAL.md

```diff
@@ -0,0 +1,253 @@
+Creating a basic S2I builder image
+---
+
+### Getting started
+
+The builder image that you specify must contain all required libraries and tools to be able to build and run the intended application.
+This tutorial uses the [openshift/base-centos7](https://github.com/openshift/sti-base) image that has, among other pre-installed libraries and tools, the epel repository added which allows us to install Nginx, since it does come with the standard centos 7 repositories.
+The tool comes with a bootstrap command that creates the directory structure and needed files to get started.
+
+`s2i create <image name> <destination directory>` - [command help](https://github.com/openshift/source-to-image/blob/master/docs/cli.md#s2i-create)
+
+`s2i create nginx-centos7 s2i-nginx`
+
+This will create a directory structure for our image and generate the artifacts, injecting the name of the desired image inside them, in our case `nginx-centos7`. This helps streamline the process of building and running the image.
+
+Directory skeleton:
+
+* `s2i-nginx`
+  * `Dockerfile` – standard Dockerfile where we’ll define the builder image
+  * `Makefile` – helper script for building and testing the builder image
+  * `test`
+    * `run` – test script, testing if the builder image works correctly
+    * `test-app/ ` – directory for your test application
```

Comment:
> are you suggesting we should get rid of it from s2i create or just from this tutorial? If the latter 

just getting rid of it from this tutorial.

> I'd also omit incremental builds, which means leaving out save-artifacts

i debated that, but i think providing one still makes sense.  again the idea here is not "tutorial" it's "skeleton image that they can fill in their own logic" so we should give them all the skeleton bits they need.  If they choose not to add any logic to the save-artifacts script, it won't hurt anything, but it'll be there if they want to.